lib/EphemeralSocket.js

```diff
@@ -105,7 +105,7 @@ ephemeralSocket.prototype.send = function (data) {
 /*
  * Send data.
  */
-ephemeralSocket.prototype._writeToSocket = function (data) {
+ephemeralSocket.prototype._writeToSocket = function (data, cb) {
```

Comment:
@Raynos if `cb` is not passed in, it gets `undefined`, correct? passing `undefined` to socket.send() down below seems fine to me. How would it break existing code?